packages/manager/modules/web-paas/src/components/additional-option/controller.js

```diff
@@ -0,0 +1,26 @@
+import get from 'lodash/get';
+
+export default class WebPaasProjectAdditionalOptionCtrl {
+  $onInit() {
+    this.disabled = false;
+  }
+
+  getPrice() {
+    const { price } = get(this.option, 'prices').find(({ capacities }) =>
+      capacities.includes('renew'),
+    );
+    if (this.option.planCode === 'storage') {
+      const stagingQuantity =
+        this.project?.totalEnvironment() ||
+        this.plan.addons?.find(
+          (addon) => addon.family === 'staging_environment',
+        ).quantity;
+      return `${(2 + stagingQuantity) *
+        price.value *
+        this.option.quantity} ${price.text.split('').pop()}`;
+    }
+    return `${price.value * (this.option.quantity || 0)} ${price.text
+      .split('')
+      .pop()}`;
```

Comment:
Is this to get currency ? This won't work as sometimes it is before the value and sometimes after 
(e.g `£5` and `5€`)